bee-protocol/src/workers/propagator.rs

```diff
@@ -52,7 +52,7 @@ async fn propagate<B: StorageBackend>(
             }
 
             // Note: There are two types of solidity carriers:
-            // * solid messages (with available history)
+            // * solid messages (with available history)(ab2
```

Comment:
What ?

Reply:
My keyboard has a habit of occasionally spewing random letters into things. I'm not sure why, it's really annoying.